app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java

```diff
@@ -210,7 +218,7 @@ private static MetricsQuery useAvailableRollups(
         newQueryBuilder.setMetrics(originalQuery.getMetrics().stream().map(metric -> {
             // Check to see if there are any rollups for this metrics
             final String metricName = metric.getName();
-            if (metricName.endsWith("_!")) {
+            if (metricName.endsWith(ROLLUP_OVERRIDE)) {
                 metrics.incrementCounter("kairosService/useRollups/bypass", 1);
```

Comment:
Should we log a `0` against `bypass` in the `else` branch?

Reply:
Fixed